src/staticanalysis/bot/static_analysis_bot/revisions.py

```diff
@@ -223,7 +232,7 @@ def namespaces(self):
         ]
 
     def __repr__(self):
-        return self.diff_phid
+        return self.diff_phid or 'unknown'
```

Comment:
Shouldn't diff_phid always have a value?

Reply:
Yes, i did that before moving the logic in the constructor